### ISSUE TITLE
`S2Coordinator#connect()` does not work for multiplayer

### ISSUE THREAD

**Issue by boehmseb** [ocraft/ocraft-s2client#12]
Using `S2Coordinator#connect()` to connect to a running starcraft instance does not work with a multiplayer setup as both clients try to connect to the same port (the one provided with this function).

**Comment by ocraft** [comment]
For multiplayer setup of this kind you need two running starcraft instances each with different open port and two different client program. Each client connects to their own game instance and must setup multiplayer port configuration that starts with some common port, for example:

### first client
```java
 S2Coordinator s2Coordinator = S2Coordinator.setup()
                .loadSettings(args)
                .setParticipants(
                        S2Coordinator.createParticipant(Race.TERRAN, bot01, "Ocraft_Bot01"))
                .connect("127.0.0.1", 5000);
                
s2Coordinator.setupPorts(2, () -> 5002, false);
s2Coordinaror.startGame();
```

### second client
```java
 S2Coordinator s2Coordinator = S2Coordinator.setup()
                .loadSettings(args)
                .setParticipants(
                        S2Coordinator.createParticipant(Race.ZERG, bot02, "Ocraft_Bot02"))
                .connect("127.0.0.1", 5001);
                
s2Coordinator.setupPorts(2, () -> 5002, false);
s2Coordinaror.joinGame();
```

Moreover only one of the clients should create the game but both of them must join to the game.

It's not a bug but this use case right now is hidden from the api user and should be better documented (right now practically everybody uses api on windows or with the ladder - both cases are automatic).

**Comment by boehmseb** [comment]
Thanks for the idea with using 2 S2Coordinators, I hadn't thought of that.

**Comment by ocraft** [comment]
Fixed in -> Linux fixes #13
